boards/arm/samv7/common/include/board_hsmci.h

```diff
@@ -59,7 +59,7 @@ extern "C"
  ****************************************************************************/
 
 int sam_hsmci_initialize(int slotno, int minor, gpio_pinset_t cdcfg,
-                         int cdirq);
+                         int cdirq, bool cdinvert);
```

Comment:
I'm just wondering of this solution as adding define `BOARD_HSMCI_CD_STATE` to `board.h` seems to me a chipper solution.
@Cynerd do you have any objections if I will rework the code and remove parameter + new data in favour of CD "inserted" level define in `board.h`?

Reply:
I am ok with that.

Honestly, that was my original version and idea. I ended up on this because the original author probably expected support for multiple SD card slots; that was my explanation for passing card detection pin. In such case, the assumption, when using macro, is that everery slot on the board is wired the same way. I do not have nothing against that assumption. I also not see a board with more than one slot in my vicinity. I only implemented it this way to keep it consistent with already established approach of passing card detection configuration.

In short, go ahead.

Reply:
Yeah. If the different slots have different CD level routing on PCB (I can't imagine why that might happen, but still) then define in board.h is definitely a limitation. I think that the issue appeared when I moved code to the common folder, so the CD level was moved out from board specific.
Seems that the current approach is more flexible, let's stick with that for now.